src/main/java/com/baro/archive/application/dto/ArchiveUnitResult.java

```diff
@@ -0,0 +1,43 @@
+package com.baro.archive.application.dto;
+
+import com.baro.archive.domain.Archive;
+import com.baro.archive.domain.ArchiveTab;
+
+public record ArchiveUnitResult(
+        Long archiveId,
+        String tabName,
+        String categoryName,
+        String content,
+        Integer copiedCount,
+        Integer savedCount
+) {
+
+    public static ArchiveUnitResult of(Archive archive) {
+        if (archive.getTemplate() == null) {
```

Comment:
archive에 메서드로 표현해도 좋아보여요
```suggestion
        if (archive.isMemo()) {
```

Reply:
오! 감사합니다 적용해놓을게요